lib/src/constants.dart

```diff
@@ -64,48 +64,48 @@ class Constants {
     "पच्पन्न",
     "छपन्न",
     "सन्ताउन्न",
-    "अन्ठाउँन्न",
-    "उनान्न्साठी",
+    "अन्ठाउन्न",
+    "उनन्साठी",
     "साठी",
-    "एकसाठी",
-    "बासाठी",
-    "तीरसाठी",
-    "चौंसाठी",
-    "पैसाठी",
-    "छैसठी",
-    "सत्सठ्ठी",
-    "अर्सठ्ठी",
+    "एकसठ्ठी",
+    "बैसठ्ठी",
+    "त्रिसठ्ठी",
+    "चौँसठ्ठी",
+    "पैँसठ्ठी",
+    "छैसठ्ठी",
+    "सड्सठ्ठी",
+    "अठ्सठ्ठी",
     "उनन्सत्तरी",
-    "सतरी",
+    "सत्तरी",
     "एकहत्तर",
     "बहत्तर",
     "त्रिहत्तर",
     "चौहत्तर",
     "पचहत्तर",
-    "छहत्तर",
-    "सत्हत्तर",
-    "अठ्हत्तर",
-    "उनास्सी",
-    "अस्सी",
+    "छयहत्तर",
+    "सतहत्तर",
+    "अठहत्तर",
+    "उनासी",
+    "असी",
     "एकासी",
     "बयासी",
-    "त्रीयासी",
+    "त्रियासी",
     "चौरासी",
     "पचासी",
     "छयासी",
     "सतासी",
     "अठासी",
     "उनान्नब्बे",
     "नब्बे",
-    "एकान्नब्बे",
-    "बयान्नब्बे",
-    "त्रियान्नब्बे",
-    "चौरान्नब्बे",
-    "पंचान्नब्बे",
-    "छयान्नब्बे",
-    "सन्तान्‍नब्बे",
-    "अन्ठान्नब्बे",
```

Comment:
all from 'एकान्नब्बे' to 'अन्ठान्नब्बे' except 'पंचान्नब्बे' are already correct here.
'पंचान्नब्बे' can be converted to 'पन्चान्नब्बे'.